src/components/AddItems.jsx

```diff
@@ -27,16 +27,23 @@ export function AddItems() {
 			const daysUntilNextPurchase =
 				event.target.elements['purchase-date'].value;
 
-			if (itemName === '') {
```

Comment:
Because this check for an empty item was removed, I was actually still able to add in an empty item.

You should consider adding in the empty check back in, so we can prevent empty items being added
<img width="986" alt="Screen Shot 2024-09-13 at 12 11 18 PM" src="https://github.com/user-attachments/assets/3e7802f7-ef34-4671-94e0-75d9dddb2c3e">